lib/qa_topic_edits.rb

```diff
@@ -155,6 +179,14 @@ def voted
     scope.current_user && ::Topic.voted(object.topic, scope.current_user)
   end
 
+  def vote_count
+    scope.current_user && ::Topic.vote_count(object.topic, scope.current_user)
```

Comment:
For this one we want to return a number rather than a boolean, so we can get rid of ``scope.current_user &&``. It should just be

```
Topic.vote_count(object.topic, scope.current_user)
```